src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -145,14 +146,16 @@ export class ExposureNotificationService {
   }
 
   async updateExposureStatusInBackground() {
-    await this.loadExposureStatus();
-    try {
-      captureMessage('updateExposureStatusInBackground', {exposureStatus: this.exposureStatus.get()});
-      await this.updateExposureStatus();
-      await this.processNotification();
-      captureMessage('updatedExposureStatusInBackground', {exposureStatus: this.exposureStatus.get()});
-    } catch (error) {
-      captureException('updateExposureStatusInBackground', error);
+    if (this.shouldPerformExposureCheck()) {
```

Comment:
Would prefer to do an "early return" here and avoid the "long" if statement

if(!this.shouldPerformExposureCheck()) return